src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -593,6 +593,9 @@ class DocBaseViewer extends BaseViewer {
             case ']':
                 this.nextPage();
                 break;
+            case 'Escape':
+                this.exitAnnotationMode();
```

Comment:
Should this handler be managed by the AnnotationControls component, instead? Otherwise we'll have to reimplement it in each viewer.